custom_components/tuya_local/devices/probreeze_pbd23w_dehumidifier.yaml

```diff
@@ -0,0 +1,111 @@
+name: Pro Breeze D-23 Dehumidifier
```

Comment:
Top level name should be without branding (all branding should be under products).